modnet/model_presets/presets.py

```diff
@@ -1,33 +1,40 @@
-batch_sizes = [64]
+def gen_presets(n_feat,n_samples):
 
-learning_rates = [0.02, 0.005]
+    batch_sizes = [32, 64]
+    learning_rates = [0.01, 0.005]
+    epochs = [500]
+    losses = ["mae"]
+    activations = ["elu"]
 
-archs = [
-    (350, [[64], [8], [8], []]),
-    (350, [[256], [128], [8], [8]]),
-    (1000, [[256], [128], [8], []]),
-]
+    if n_feat<500: # For compounds with limited knowledge, e.g. composition only
+        n_feat_list = [60, 110]
+    else:
+        n_feat_list = [320, 520]
```

Comment:
I think something like this might work better? We might want to be careful about someone trying to make a model from very few features too...

```suggestion
n_feat_list = (min(0.2 * n_feat, 50), max(0.5 * n_feat, 500))
```

Reply:
(I'll push this but we can discuss)

Reply:
ok for me !

Reply:
or could add it to n_feat_list (if it doesn't make it to much to compute) => the chosen values (60,110,320,520) were often occurring in our tests.